src/components/NumericalFacets.tsx

```diff
@@ -0,0 +1,100 @@
+import { FacetsProvider, RangeInput, RangeInputCssClasses } from './Filters';
+import { FilterGroup, FilterGroupCssClasses } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+import { DisplayableFacet, NumberRangeValue } from '@yext/answers-headless-react';
+import { StandardFacetsProps } from './StandardFacets';
+import { isNumberRangeFilter } from '../models/NumberRangeFilter';
+import { useMemo } from 'react';
+
+/**
+ * The CSS class interface for {@link NumericalFacets}.
+ *
+ * @public
+ */
+export interface NumericalFacetsCssClasses extends FilterGroupCssClasses, RangeInputCssClasses {
+  container?: string,
+  divider?: string,
+  rangeInputContainer?: string,
+  rangeInputDivider?: string
+}
+
+/**
+ * Props for the {@link NumericalFacets} component.
+ *
+ * @public
+ */
+export interface NumericalFacetsProps extends StandardFacetsProps {
+  inputPrefix?: JSX.Element,
+  getFilterDisplayName?: (numberRange: NumberRangeValue) => string,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: NumericalFacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays simple facets applicable to the current vertical search.
+ *
+ * @remarks
+ * Numerical facets is not included. Hierachical facets will not be display in a
+ * tree level structure. Use `hiddenFields` to exclude hierachical facets, if any,
+ * when using this component.
+ *
+ * @param props - {@link NumericalFacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function NumericalFacets({
+  searchOnChange,
+  hiddenFields = [],
+  getFilterDisplayName,
+  inputPrefix = <>$</>,
+  customCssClasses = {},
+  ...filterGroupProps
+}: NumericalFacetsProps) {
+  const rangeInputCssClasses = useMemo(() => {
+    return {
+      ...customCssClasses,
+      container: customCssClasses.inputContainer,
+      divider: customCssClasses.divider
```

Comment:
is this needed if the name is staying the same?

Reply:
this should be:
`container: customCssClasses.rangeInputContainer,
 divider: customCssClasses.rangeInputDivider`
 since there's css name conflict between this component and the RangeInput one